pkg/osquery/tables/wifi_networks/wifi_networks.go

```diff
@@ -0,0 +1,149 @@
+package wifi_networks
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/kolide/launcher/pkg/osquery/tables/wifi_networks/internal"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+//go:generate go-bindata -nometadata -nocompress -pkg internal -o internal/assets.go internal/assets/
+
+type execer func(ctx context.Context, buf *bytes.Buffer) error
+
+type WlanTable struct {
+	client    *osquery.ExtensionManagerClient
+	logger    log.Logger
+	tableName string
```

Comment:
Only needed if you're meta-programming multiple tables. No point to having the variable here.